bin/BuildPackages.sh

```diff
@@ -130,8 +109,12 @@ notice \
 "Note that many GAP packages require extra programs to be installed," \
 "and some are quite difficult to build. Please read the documentation for" \
 "packages which fail to build correctly, and only worry about packages" \
-"you require!" \
-"Logging into ./$LOGDIR/$LOGFILE.log"
+"you require!"
+
+echo_run() {
+  echo "$@"
```

Comment:
This does exactly what it name suggests: It echos a command before running it. This is a limited emulation of what set -x / -v do; it is used to output the exact configure command that is being run, which is useful for the log and for debugging build issues. It allows you to see which flags were passed to configure exactly. It is meant to stay.

Sure, this could use `notice` or add the prefix "Running " but personally I wouldn't find it  useful. A matter of taste, I guess